metron-platform/metron-parsers/README.md

```diff
@@ -337,6 +337,28 @@ The following config will rename the fields `old_field` and `different_old_field
                       ]
 }
 ```
+* `REGEX_SELECT` : This transformation lets users set an output field to one of a set of possibilities based on matching regexes. This transformation is useful when the number or conditions are large enough to make a stellar language match statement unwieldy.
+ 
+The following config will set the field `my_output` to one of the
+following, dependent upon the value of the `my_input` field:
+* `awesome` if `my_input` starts with `metron` or `mortron`
+* `boo` if `my_input` starts with `scary`
+```
+{
+...
+    "fieldTransformations" : [
+          {
+            "transformation" : "REGEX_SELECT"
+          , "input" : "my_input"
+          , "output" : "my_output"
+          , "config" : {
+            "awesome" : [ "^metron.*", "^mortron.*" ],
```

Comment:
A little nit.  It makes it harder to grok the docs when you use non-sensical phrases like 'boo', 'scary', etc in the primary example.  Is there an actual use case that we could show here instead?

Reply:
Same goes for the test cases, IMO, but I am not going to hold up the PR for that.

Reply:
I changed the parsers docs to reflect the use-case that I wrote up for the next PR.  That should fold in well together and be a real use-case.  Thanks for the feedback.